packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -33,6 +33,8 @@ abstract class RenderToggleable extends RenderConstrainedBox {
     required Color inactiveColor,
     Color? hoverColor,
     Color? focusColor,
+    Color? reactionColor,
```

Comment:
Should there be any tests added/updated for the changes to this class?

Reply:
I could see a test being added for the lerping of the colors, otherwise I am testing the behavior within the Checkbox, Radio and Switch tests. Let me try to add a test for the lerping of colors.